code/src/entrance.c

```diff
@@ -28,6 +28,14 @@ void Scene_Init(void) {
 
     gRestrictionFlags[72].flags3 = 0; // Allows farore's wind in GTG
     gRestrictionFlags[94].flags3 = 0; // Allows farore's wind in Ganon's Castle
+    
+    //Allow Farore's Wind anywhere if given by settings
+    if (gSettingsContext.faroresWindAnywhere){
+        for (int i = 0; i < 99; i++){
+			gRestrictionFlags[i].flags3 &= ~0x30;
+			gRestrictionFlags[i].flags3 &= ~0x03;
```

Comment:
This line makes it possible to use every item anywhere, not only FW. Is that intentional?